models/user_cgn.ts

```diff
@@ -0,0 +1,48 @@
+import { Container } from "@azure/cosmos";
+import {
+  CosmosdbModelVersioned,
+  RetrievedVersionedModel
+} from "io-functions-commons/dist/src/utils/cosmosdb_model_versioned";
+import { wrapWithKind } from "io-functions-commons/dist/src/utils/types";
+import * as t from "io-ts";
+import { FiscalCode } from "italia-ts-commons/lib/strings";
+import { CgnStatus } from "../generated/definitions/CgnStatus";
+
+export const USER_CGN_COLLECTION_NAME = "user-cgns";
+export const USER_CGN_MODEL_PK_FIELD = "fiscalCode";
```

Comment:
Other commons models have `as const` casting for `PK` field name.
```suggestion
export const USER_CGN_MODEL_PK_FIELD = "fiscalCode" as const;
```